assignment-1/time_pipe.c

```diff
@@ -0,0 +1,59 @@
+#include <sys/types.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <iostream>

```

Comment:
We're using C for this assignment, and `<iostream>` is a C++ library. Not sure if he mentioned C++ being allowed, but if it is, then the file extension should be changed to .cpp.